test/files/pos/t10223.scala

```diff
@@ -0,0 +1,13 @@
+// scalac: -Xsource:3
```

Comment:
Not necessary?

Reply:
but I did think that it would be useful to at least warn if override (by abstract member) behavior changes, under `-Xsource;3`.